### ISSUE TITLE
What is this?

### ISSUE THREAD

**Issue by benrap** [GhostFlying/force-metered-ap#1]
Sorry if i sound like a dummy, but I'm not a programmer and would like to know what all the AP stuff means? What does this xposed module do?

**Comment by GhostFlying** [comment]
To make the third-party apps obey the metered AP settings so they will not
waste my data traffic.

**Comment by Droidphilev** [comment]
I don't understand this either. How can an app use more data than the limit that the user sets up in settings?

**Comment by GhostFlying** [comment]
No, the app can not limit the data usage, it only let some apps feels they
are under the cellular network even the device is connected to Wi-Fi.

**Comment by Droidphilev** [comment]
ah ok. In other words:  the data limit/counter for cellular network will also work for selected apps while on wifi?

**Comment by GhostFlying** [comment]
No, the data limit will not work, it only let apps treat your device is not
connected to WiFi so they may behavior different.

Like telegram, it can be configured download media automatically only under
WiFi, but if you connect to one metered AP, it will download automatically
too. This module will tell it your device is not connected to any WiFi AP
so make it not download automatically.

**Comment by Droidphilev** [comment]
Ok, it's clear now. Thanks for the info!

**Comment by benrap** [comment]
Ok thanks guys, i get it. It tricks specific apps into thinking you are connected/ not connected to wifi. Thanks to anyone who commented!!
